compiler/rustc_borrowck/src/lib.rs

```diff
@@ -803,7 +809,11 @@ impl<'cx, 'tcx, R> rustc_mir_dataflow::ResultsVisitor<'cx, 'tcx, R> for MirBorro
                 }
             }
 
-            TerminatorKind::Resume | TerminatorKind::Return | TerminatorKind::GeneratorDrop => {
+            // FIXME(explicit_tail_calls): do we need to do something similar before the tail call?
```

Comment:
what's this asking? Handling it like a return Seems correct to me.

Reply:
I do not remember what I meant by that...

Handling it like a return does seem correct, so I just removed the fixme.